python/lsst/meas/base/plugins.py

```diff
@@ -71,6 +73,72 @@
                     TransformClass=bl.ApertureFluxTransform, executionOrder=BasePlugin.FLUX_ORDER)
 
 # --- Single-Frame Measurement Plugins ---
+class SingleFrameFocalPlaneConfig(SingleFramePluginConfig):
+    pass
+
+@register("base_FocalPlane")
+class SingleFrameFocalPlanePlugin(SingleFramePlugin):
+    '''
+    Algorithm to calculate position on focal plane
+    '''
+
+    ConfigClass = SingleFrameFocalPlaneConfig
+
+    @classmethod
+    def getExecutionOrder(cls):
+        return cls.APCORR_ORDER
```

Comment:
Why `APCORR_ORDER`?

It looks like this requires a centroid but not a shape, so I guess `SHAPE_ORDER` would work.